server/src/db/LawyerSeeder.js

```diff
@@ -0,0 +1,25 @@
+import { Lawyer } from "../models/index.js";
+class LawyerSeeder {
+  static async seed() {
+    const lawyerData = [
+      {
+        name: "Jane",
+        id: "3",
+        url: "http",
+        location: "NYC",
+        language: "Chinese",
+        educationBackground: "Master",
+        workExperience: "years",
+      },
+    ];
+
+    for (const lawyer of lawyerData) {
+      const lawyerObject = await Lawyer.query().findOne({ id: lawyer.id });
```

Comment:
should search based on information we know, like the name of the lawyer `.findOne({ name: lawyer.name })`